internal/router/router_test.go

```diff
@@ -0,0 +1,113 @@
+package router
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/storage"
+	"github.com/julienschmidt/httprouter"
+	"github.com/stretchr/testify/assert"
+)
+
+// TODO(chmouel): Implement a memory store which could be used between all tests instead of having to mock it everywhere
```

Comment:
I agree, but why don't you create a GitHub issue to track this, instead of a TODO in the code?